contracts/Marionette.sol

```diff
@@ -137,7 +150,15 @@ contract Marionette is IMarionette, AccessControlEnumerableUpgradeable {
 
         if (target.isContract()) {
             if (data.length >= 4) {
-                return target.functionCallWithValue(data, value);
+                if (_getHashedSelector(data) == _multiSendHashed()) {
+                    // slither-disable-next-line controlled-delegatecall,low-level-calls
+                    (bool success, bytes memory result) = target.delegatecall(data);
```

Comment:
Let's use functionDelegateCall by Openzeppelin so we do not need to handle errors

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Address.sol#L170